src/wallet/coinselection.cpp

```diff
@@ -68,9 +68,9 @@ struct {
  *
  * waste = selectionTotal - target + inputs × (currentFeeRate - longTermFeeRate)
  *
- * The algorithm uses two additional optimizations. A lookahead keeps track of the total value of
+ * The algorithm uses two additional optimizations. A lookahead (tk) keeps track of the total value of
  * the unexplored UTXOs. A subtree is not explored if the lookahead indicates that the target range
- * cannot be reached. Further, it is unnecessary to test equivalent combinations. This allows us
+ * cannot be reached. Further, it is unnecessary to test equivalent combinations (tk). This allows us
```

Comment:
In 9580c0e49030f13fe57603521b59b96e6e50cf82 "coinselection: rewrite BnB in CoinGrinder-style"

What is `(tk)` mean?